models/sessions/scratch/snowplow_mobile_sessions_aggs.sql

```diff
@@ -13,6 +13,7 @@
 
 with events as (
   select
+    es.session_identifier,
```

Comment:
What's the reason for adding this in here? We overwrite the session_id field anyway and it's in the main session table anyway?